graphql/util.go

```diff
@@ -574,13 +574,40 @@ func isTaskBlocked(ctx context.Context, at *restModel.APITask) (*bool, error) {
 	return &isBlocked, nil
 }
 
+func isExecutionTask(ctx context.Context, at *restModel.APITask) (*bool, error) {
+	t, err := task.FindOneId(*at.Id)
```

Comment:
Instead of doing this, you should call at.BuildFromService. All the data is already there, it just needs to be converted